tests/network/network_functions_test.cpp

```diff
@@ -2,6 +2,6 @@
 #include "gtest/gtest.h"
 
 TEST(NetworkFunctions, urlEncode) {
-  ASSERT_EQ(cuhksz::urlEncode("http://google.com®"), "http%3A%2F%2Fgoogle.com%C2%AE");
-  ASSERT_EQ(cuhksz::urlEncode("Hello Günt+er"), "Hello%20G%C3%BCnt%2Ber");
+  ASSERT_EQ(cuhksz::urlEncode(u8"http://google.com®"), "http%3A%2F%2Fgoogle.com%C2%AE");
```

Comment:
sorry for the confusing, the `u8` should be deleted here, otherwise not right on windows, i don't know why